gnocchi/tests/functional/gabbits/resource-type.yaml

```diff
@@ -768,8 +768,6 @@ tests:
             min_length: 0
             max_length: 255
       status: 400
-      response_strings:
```

Comment:
This is actually removing a useful test. Did the error reported changed?

Reply:
Yes, changed the output. Will revert and check output again.